build.gradle

```diff
@@ -1,6 +1,7 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
+    ext.kotlin_version = '1.6.0-M1'
```

Comment:
`1.6.0-M1` is [an early access version](https://kotlinlang.org/docs/eap.html#build-details) of the Kotlin plugin. We use stable version for all our tooling, so I'd suggest using version `1.5.31`.

Reply:
That said, let's not add a new line to add this to `ext` when can simply add to the array below.
```suggestion
```

Reply:
Additionally, we use `camelCase` for naming all our variables, so this should end up looking like the following in the `ext` array below:

```groovy
        kotlinGradlePluginVersion = "1.5.31"
```

Reply:
> Hi @DeFuex! Thanks a lot for your contribution. I have reviewed it and added some comments. As I am not a Kotlin expert, I have a question as well: what is the reason we might want to add the Kotlin plugin to the Sample Chat App's `build.gradle`? Is it so the project can also use Kotlin in the source code instead of only Java?
> 
> I ask this because we are not looking to support Kotlin in the Azure SDK for Android for the moment. You can use our libraries with Kotlin applications, but Java is the SDK's programming language of choice.

Hi :), before i change anything here according to the review (thanks by the way), there's a few things to consider trying out Kotlin at least once https://kotlinlang.org/docs/comparison-to-java.html#what-java-has-that-kotlin-does-not . I'm not saying you ultimately should use it if you are fine with Java, but for what i've seen in the recent projects i was working with it, it reduces quite some boilerplate or makes the flow for the development easier in general. It leans on Java, but it offers a concise set of functionality that is just there and nice to use. And it also compiles down to the JVM, so you can mix java and kotlin classes and slowly migrate or at least try it out to see for yourself. At my work (banking environment) we are using it for Android and Backend and it has proven itself for the last years. Somewhere in the next years we might get java version >16, which will conclude same functionalities as Kotlin does. Also interesting for this topic -> https://youtu.be/te3OU9fxC8U

Reply:
Thanks for the info Timo :) While I personally love the idea of working with Kotlin, using it in the Azure SDK for Android does not align with our current roadmap. We did spend quite a good while debating between using Java and Kotlin and are aware of the pros/cons, but ultimately decided to go with Java since it was better suited for our current needs. This does not mean that we are completely discarding the idea of supporting Kotlin in the SDK, it just might be a while before we reach that point.